Adaptors/Amqp/src/SessionAmqp.cs

```diff
@@ -1,164 +0,0 @@
-// This file is part of the ArmoniK project
```

Comment:
Where the functionality of this file went?

Reply:
new Session(connection); No need for that complexity